src/test/java/api/queue/RequestQueueResourceTest.java

```diff
@@ -223,6 +231,127 @@ void refuseAttemptToMoveRequestBeingFulfilledFromOneOfTheTopPositionsWhenTlrEnab
       is("Requests can not be displaced from top positions when fulfillment begun."));
   }
 
+  @Test
+  void shouldGetRequestQueueForItemSuccessfully() {
```

Comment:
Regarding code smell rules, the test method should not contain too many assertions. So refactor this method to reduce the number of assertions from 33 to less than 25.

Reply:
fixed

Reply:
fixed